styles.css

```diff
@@ -383,3 +385,104 @@ li a {
   margin-bottom: 8px;
   margin-right: 8px;
 }
+
+.form {
+  background-image: url("images/illustration-contact-form.svg");
+  background-position: right bottom;
+  background-repeat: no-repeat;
+}
+
+.form h2 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  padding: 0 24px;
+  margin-bottom: 5%;
+}
+
+.contact_form {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.form-inputs {
+  padding-left: 0;
+  width: 70%;
+}
```

Comment:
- [ ] Please increase the width of your forms fields to match the figma template, notice how it is taking around 90% of the viewport width in the design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/220649691-f2489dd1-c0cb-4ca8-8c4e-e0c59f030771.png)

your design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/220649864-cd7a54a9-87cb-4ecf-ab35-ae7f1140a0fc.png)